test/protocol/ProtocolDiamondTest.js

```diff
@@ -946,7 +946,7 @@ describe("ProtocolDiamond", async function () {
     });
 
     context("💔 Revert Reasons", async function () {
```

Comment:
I'm supplying suggestions below for dropping the "Should revert if..." from these statements to make the style match the rest of our unit tests, which simply supply the condition, since they are listed in an explicit Revert Reasons context.

Reply:
done